src/core/control/Control.cpp

```diff
@@ -1608,6 +1651,11 @@ void Control::openFileWithoutSavingTheCurrentDocument(fs::path filepath, bool at
         return;
     }
 
+    if (Util::hasPngFileExt(filepath)) {
+        callback(openPngFile(fs::path(filepath), attachToDocument, scrollToPage));
```

Comment:
For consistency and for avoiding a copy:
```suggestion
        callback(this->openPngFile(std::move(filepath), attachToDocument, scrollToPage));
```